autoload/lsp/ui/vim.vim

```diff
@@ -287,9 +287,15 @@ function! s:handle_location(ctx, server, type, data) abort "ctx = {counter, list
                 execute l:cmd . ' | call cursor('.l:loc['lnum'].','.l:loc['col'].')'
                 redraw
             else
-                call setqflist(a:ctx['list'])
+				let l:cwd = getcwd() . "/"
+				let l:ctxlist = luaeval("require('lsp-file').handle_loc_list(_A.data, _A.cwd)", {
+							\ "data": a:ctx['list'],
+							\ "cwd": l:cwd,
+							\ })
+                call setloclist(0, l:ctxlist)
```

Comment:
some people perform quickfix, some loclist. We can't change the default behavior now. should be under a config value.